wled00/cfg.cpp

```diff
@@ -153,14 +153,21 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
     }
   } else {
     // new install/missing configuration (button 0 has defaults)
-    if (fromFS)
-      for (uint8_t s=1; s<WLED_MAX_BUTTONS; s++) {
+    if (fromFS) {
+      // relies upon only being called once with fromFS == true, which is currently true.
+      uint8_t s = 0;
+      int8_t pin = btnPin[0]; // initialized to #define value BTNPIN, or zero if not defined(!)
```

Comment:
No need for new variable (unnecessary stack & CPU cycle use).
Otherwise another good catch!

Reply:
Sorry, habits from years of defensive coding, where user-supplied buffers were potentially maliciously modified between time-of-check, and time-of-use.  Removed new variable in 42993e8.